File: 2015 czerwiec/c++/1.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main()
{
    ifstream in("kody.txt");
    ofstream out("kody1.txt");

    for (int linia = 1; linia <= 500; linia++)
    {
        string s;
        in >> s;
        int even = 0, odd = 0;
        for (int i = 0; i < s.size(); i++)
        {
            if ((i + 1) % 2)
                odd += s[i] - '0';
            else
                even += s[i] - '0';
        }
        out << even << ' ' << odd << '\n';
    }    
}

File: 2015 czerwiec/python/1.py
with open("kody.txt", "r") as f:
    k = f.read().strip().split('\n')

with open("kody1.txt", "w") as f:
    for liczba in k:
        even, odd = 0, 0
        for i in range(len(liczba)):
            if (i + 1) % 2:
                odd += int(liczba[i])
            else:
                even += int(liczba[i])
        f.write(str(even) + ' ' + str(odd) + '\n')